Snakefile

```diff
@@ -185,6 +185,22 @@ if config['enable'].get('retrieve_natura_raster', True):
         run: move(input[0], output[0])
 
 
+if config['enable'].get('build_ship_raster', False):
+    rule build_ship_raster:
+        input:
+            ship_density="data/bundle/shipdensity/shipdensity_global.zip",
```

Comment:
add this file to `datafiles = [..., "shipdensity/shipdensity_global.zip"]` further up.